### ISSUE TITLE
[BUG]: Contents of `Toast` component are not announced in screen reader

### ISSUE THREAD

**Issue by o-hook** [porsche-design-system/porsche-design-system#3338]
Scope
---
Though the `Toast` component was developed with accessibility best practices in mind (including the usage of `role="status"`, the contents of the rendered `Toast` component are not announced in screen reader. This is due to the insufficient implementation of `role="status"` attribute.

Out of Scope
---

Notes  
---
- ARIA live attributes should always be rendered initially to have the best support for assistive technologies.
- Possible solution: try to add `role="status"` to the `host` element or with `ElementInternals` API

Acceptance criteria
---
- [ ] Contents of `Toast` component get announced in screen reader when host is rendered initially.

Subtasks
---
- [ ] Task

**Comment by o-hook** [comment]
Will be fixed with #3130
